biosimulations/apps/dispatch-service/src/app/app.controller.spec.ts

```diff
@@ -137,5 +140,15 @@ describe('AppController', () => {
     });
   });
 
-  
+  describe('test jobMonitorCronJob', () => {
+    it('should return promise', () => {
+      const appController = app.get<AppController>(AppController);
```

Comment:
This test is going to break if the job with that id is removed from the cluster.